tools/model/import/debug-info/CMakeLists.txt

```diff
@@ -4,5 +4,5 @@
 
 revng_add_executable(revng-model-import-debug-info Main.cpp)
 
-target_link_libraries(revng-model-import-debug-info revngModel
+target_link_libraries(revng-model-import-debug-info revngABI revngModel
```

Comment:
This tool was using ABI without linking to it? 🤔. No idea how that was even working.

@aleclearmind, feel free to close this if the commit looks good.

Reply:
Tis entire file.